amsterdam-email-api/backend/plugin/src/main/kotlin/com/ritense/valtimo/amsterdam/emailapi/plugin/EmailApiPlugin.kt

```diff
@@ -61,6 +58,8 @@ class EmailApiPlugin(
     )
     fun sendEmail(
```

Comment:
Graag in de plugin actie de beschrijving en titel uitbreiden dat het een email is met optioneel zaakId en relatiecode